runtime/flang/getarg.c

```diff
@@ -27,13 +27,14 @@ __INT_T
 ENTRY(IARGC, iargc)() { return __io_get_argc() - 1; }
 
 void
-ENTRY(GETARGA, getarga)(__INT_T *n, DCHAR(arg) DCLEN64(arg))
+ENTF90(GETARGA, getarga)(__INT_T *n, DCHAR(arg), __INT_T *int_kind DCLEN64(arg))
```

Comment:
Disregard my previous comment. However, what is your reason the for the switch from ENTRY to ENTRYF90?

Reply:
To pass the additional argument `int_kind` to the runtime, this patch creates a predeclared subroutine for `GETARG`, and as a result, the prefix `ENTRY` cannot be used anymore. `ENTF90` is used here to mimic the code for `GET_COMMAND_ARGUMENT`.